app/models/site.rb

```diff
@@ -67,6 +67,13 @@ def full_address
     address.select(&:present?).join(', ')
   end
 
+  def confirm_full_address
```

Comment:
in app/views/support/schools/confirm.html
```
component.with_row do |row|
              row.with_key { t("support.providers.schools.address") }
              row.with_value(text: @site.confirm_full_address)
```
it lists address lines vertically as per the prototype rather than comma separated

Reply:
Could this go in the site decorator as it's specific to the presentation? We have some other address related things in there

Reply:
but it has the same issue as the provider decorator

Reply:
I think draper allows you to just call `.decorate` on the model?